events/events-team/comms/README.md

```diff
@@ -1,55 +1,191 @@
-# Communications Lead Handbook
+# Communications Team Handbook
 
 This document defines communications activities needed to run a
-Contributor Summit.
+Kubernetes Contributor Summit (KCS). The Comms Team consists of 1-2 lead(s), shadows, volunteers, at least 1 emeritus lead, and a Website Manager.
 
-- [Overview](#overview)
-- [Skills and Qualifications](#skills-and-qualifications)
-- [Responsibilities](#responsibilities)
-  - [Communication Channels and Property Access](#communication-channels-and-property-access)
+Comms Leads are responsible for leading the comms team, training shadows, and ensuring all comms activities are planned, prioritized, and completed successfully. They are committed to run the comms team for the current cycle and advise as Emeritus KCS Comms Lead in the following comms cycle (2 cycles, 1 year). Must have [Kubernetes Org Membership]. Expected to attend in-person.
+
+Shadows will learn both how to do comms and how to run the KCS Comms team, as directed by the Comms Lead. They are committing to be involved with the event team for 1.5 years or 3 comms cycles (shadow, lead, and emeritus). Must either have [Kubernetes Org Membership] or be planning to pursue org membership.
```

Comment:
I would change "persue" to be, "on their way towards org membership"

I don't want to exclude anyone, but at the same time a shadow that's not an org member; feels like that'd be harder than expected. Have we ever granted org membership to someone that is helping with the current comms cycle? cc @jberkus

Reply:
We've given people a bump, but not if KCS Comms was their very first contribution.  So +1 on "in the process of applying for org membership".

Reply:
A couple of the comms shadows this cycle got their org membership during the cycle.